### ISSUE TITLE
Notification setting - 2 issues

### ISSUE THREAD

**Issue by licaon-kter** [mattermost/mattermost-mobile#1462]
#### Environment Information

- Device Name:  Xperia/Samsung 6.0.1
- Mattermost App Version: 1.6.1
- Mattermost Server Version: 4.7.2

#### Steps to reproduce
1. Open app, settings, notifications, email
2. If one is not selected, select one (eg. 15 minutes), save

#### Expected behavior
1. See an option selected, the one you setup earlier via the web UI
2. After selection, kill the app, open this setting, the option is selected

#### Observed behavior
No option is selected.
Every time you (kill and) open the app, then settings, there is no option selected.

Not sure if 2 issues (get notification from server + selection does not stick) or just one.

**Comment by lindy65** [comment]
Hi @licaon-kter 

Thanks for your feedback :)

This issue has been resolved... Please upgrade to the [latest version of the app](https://about.mattermost.com/download/#mattermostApps) as well as Mattermost server (v4.9 released today - April 16th).

Thanks!

**Comment by licaon-kter** [comment]
Still an issue here w/ 4.9.0 server & 1.7.1 app

No radio button is selected when I open the setting.
I check 15min, save, kill the app, reopen the app, no radio button selected.

**Comment by licaon-kter** [comment]
Huawei 8.0.0 / Samsung 6.0.1

The word `save` was mentioned in both of my posts. 😐

**Comment by lindy65** [comment]
Sorry about that @licaon-kter - I see you mentioned saving 🤦‍♀️ 

I'm wondering whether it could be an issue with email batching (you mention selecting `15 min`).  Email batching was off on the server when I tested earlier on so I had only `immediately` and `never` as options to select.

I've tested with email batching on and can _intermittently_ get one of the settings ("Immediately", "Every 15 minutes", "Every hour", "Never") to stick and save but there is definitely an issue with these particular settings so I have created [this ticket](https://mattermost.atlassian.net/browse/MM-10215) to track a fix.
